debian/rules

```diff
@@ -1,114 +1,29 @@
 #!/usr/bin/make -f
-# -*- makefile -*-
-# Sample debian/rules that uses debhelper.
-# GNU copyright 1997 to 1999 by Joey Hess.
 
 # Uncomment this to turn on verbose mode.
 #export DH_VERBOSE=1
 
-CFLAGS = -Wall -Wno-format -g
+CFLAGS_MAINT  += -Wall -Wno-format -Wno-format-security -g -DTAP_AND_HOLD
+LDFLAGS_MAINT += -Wl,--as-needed
+
+export DEB_CFLAGS_MAINT_APPEND  = $(CFLAGS_MAINT)
```

Comment:
AFAIK h-a-m is using c++ mostly

Reply:
Saying it should be CXXFLAGS?

Reply:
I don't know. I only noticed that the flags you set were not used while the package was built.

Reply:
I would check how the  build system before your porting was doing the things.

Reply:
Added CXXFLAGS in 780bdf22fd425ba5acfd175baf70609250d7907c